test/ClangImporter/cxx_interop_ir.swift

```diff
@@ -3,8 +3,8 @@
 import CXXInterop
 
 // CHECK-LABEL: define hidden swiftcc void @"$s6cxx_ir13indirectUsageyyF"()
-// CHECK: %0 = call %"class.ns::T"* @_Z5makeTv()
-// CHECK: call void @_Z4useTPN2ns1TE(%"class.ns::T"* %2)
+// CHECK: %0 = call %"class.ns::T"* @{{_Z5makeTv|?makeT@@YAPAVT@ns@@XZ}}()
```

Comment:
That `?` is suspect in a regex context…

Reply:
remote desktop over shuttle wifi = bad time :-(.  Yes, these need escaping